### ISSUE TITLE
Usage of creds parameter is deprecated in msrest service client

### ISSUE THREAD

**Issue by jfthuong** [Microsoft/azure-devops-python-api#194]
The getting started provides the following code to establish a connection
```python
# Create a connection to the org
credentials = BasicAuthentication('', personal_access_token)
connection = Connection(base_url=organization_url, creds=credentials)
```

However, while running this code, the following warning is raised:
```python
c:\python37-32\lib\site-packages\msrest\service_client.py:260: DeprecationWarning: Creds parameter is deprecated. Set config.credentials instead.
    DeprecationWarning)
```

The example in Getting Started could be updated to provide the new way of setting the credentials.

**Comment by tedchamb** [comment]
Fixed in #198

**Comment by tedchamb** [comment]
Fixed in 5.0.0b4
https://github.com/Microsoft/azure-devops-python-api/releases/tag/5.0.0b4
